### PR TITLE
Moving args to ros parameters

### LINKED ISSUE #8: Make scipts use ROS's parameter server rather than input args
Most of the packages in this repository use command line arguments rather than parameters. I believe this was done to allow more than one data capture node to run at the same time. We should be able to have multiple nodes with different parameters, if necessary.
* [ ] `aws_uploader`
* [ ] `video_capture`
* [ ] `audio_capture`

### PR DESCRIPTION
**Pull request by anandgokul18**
Moving the scripts from using args to using ROS parameter server for video_capture and audio_capture2 modules

Fixes #8

### CODE REVIEW

**Review by emilyxzhou — APPROVED** [review]
Looks good! Can you fix the missing signoff before merging?

**Review by audrow — APPROVED** [review]
Looks good. Same comment as @emilyxzhou.

**Pull request merged**
